ui/src/views/Documentation/Reference/SchemaIndex.jsx

```diff
@@ -0,0 +1,48 @@
+import React, { Component, Fragment } from 'react';
+import { string } from 'prop-types';
+import { withRouter } from 'react-router-dom';
+import sortBy from 'lodash/sortBy';
+import filter from 'lodash/filter';
+import SchemaPanel from './SchemaPanel';
+import HeaderWithAnchor from '../components/HeaderWithAnchor';
+import references from '../../../../../generated/references.json';
+
+@withRouter
+export default class SchemaIndex extends Component {
+  static propTypes = {
+    // the service name to document
+    serviceName: string.isRequired,
```

Comment:
This prop isn't passed in `ui/docs/reference/schema/index.md`.. I think it can be omitted.